src/azure-cli/azure/cli/command_modules/network/_params.py

```diff
@@ -873,8 +873,8 @@ def load_arguments(self, _):
             c.argument('watcher_name', ignore_type, validator=get_network_watcher_from_vm)
             c.ignore('location')
             c.ignore('watcher_rg')
-            c.argument('vm', help='Name or ID of the VM to target.')
-            c.argument('resource_group_name', help='Name of the resource group the target VM is in. Do not use when supplying VM ID.')
+            c.argument('vm', help='Name or ID of the VM to target. If the name of the VM is provided, the --resource-name is required.')
```

Comment:
Do you mean `--resource-group`? I don't see `--resource-name` on the test-ip-flow command.

Reply:
fix